src/content/self-hosted/install/deployment.mdx

```diff
@@ -63,31 +47,33 @@ $ helm install okteto okteto/okteto -f config.yaml --namespace=okteto --create-n
 After a few seconds, all the resources will be created. The output will look something like this:
 
 ```
-Release "okteto" has been installed. Happy Helming!
-NAME: okteto
-LAST DEPLOYED: Thu Mar 26 18:07:55 2020
-NAMESPACE: okteto
-STATUS: deployed
-REVISION: 1
-TEST SUITE: None
-NOTES:
-1. Create the following DNS record, pointing it to the NGINX controller service External-IP:
-- "*.dev.example.com"
-
-You can retrieve the External IP by running:
-    kubectl get service -l=app.kubernetes.io/name=ingress-nginx,app.kubernetes.io/component=controller --namespace=okteto
-
-2. Once you create both DNS entries you can access your Okteto instance at this URL:
-https://okteto.dev.example.com
-```
+Congratulations! Okteto is successfully installed!
 
-Once the chart is installed, follow the onscreen instructions to finalize the configuration.
+Follow these steps to access your Okteto instance:
 
-## Sign in to your Okteto instance
+1. Start a port-forward to the ingress service by running:
 
-After a successful installation, you can access your Okteto instance at `https://okteto.SUBDOMAIN`. Your account will be automatically created as part of the login process. The first user to successfully login into the instance will be automatically assigned the `administrator` role.
+   $ kubectl port-forward service/okteto-ingress-nginx-controller 8443:443 --namespace okteto
 
-> **Important**: The default installation is not recommended for production use. We highly advise following the steps outlined in the [Admin Dashboard](administration/dashboard.mdx) before giving your team access to your Okteto instance.
+2. Access your Okteto instance at https://okteto.localtest.me:8443/login#token=******
+
+Follow these steps to configure the Okteto CLI:
+
+1. Install the Okteto CLI:
+
+   $ curl https://get.okteto.com -sSfL | sh
+
+2. Configure the Okteto CLI context:
+
+   $ okteto context use https://okteto.localtest.me:8443 --token ******
+
+3. You're all set to complete our getting started guide:
 
-The default installation is all that's required to follow our [getting started](getting-started.mdx) guide.
-Just remember to replace https://cloud.okteto.com by `https://okteto.SUBDOMAIN` and you will be good to go!
\ No newline at end of file
+   https://www.okteto.com/docs/using-dev-envs/
+
+Happy coding!
+```
+
+Once the chart is installed, follow the onscreen instructions to finalize the configuration.
```

Comment:
I think we need documentation for the different steps, so that the admin knows what they need to prep for the future. I don't think we can relay just on the dashboard to convey this information. @omnomagonz what do you think?

Reply:
@rberrelleza Agreed.

I would expect to see the first mentions of these installation steps mentioned in `/self-hosted/install/preparation` where they're outlined similar to the UI. For example:

> Before installing Okteto we recommend you gather the following information and have appropriate levels of access:
> 
> 1. A domain you wish to use
> 2. Certificates to use with the Kubernetes API server
> 3. OAuth access to configure this with Okteto
> 4. External storage environment for the Okteto Registry
> 5. Dedicated Load Balancer for Buildkit

We could then write a few sentences for each to explain what they're used for (so the installer can decide if they want to skip the step or not) and provide context for why they're important for Okteto.

Then this page (`/self-hosted/install/deployment`) could have an intro statement "Be sure to follow the steps in `preparation` first", which would likely prepare them for this part of the docs.

If we're in agreement on that I can draft a PR 👍